index.js

```diff
@@ -39,11 +39,12 @@ function viewport(bounds, dimensions, minzoom, maxzoom, tileSize, allowFloat) {
         width = tr[0] - bl[0],
         height = bl[1] - tr[1],
         ratios = [width / dimensions[0], height / dimensions[1]],
-        center = [(bounds[0] + bounds[2]) / 2, (bounds[1] + bounds[3]) / 2],
-        adjusted = getAdjusted(base, ratios, allowFloat),
-        zoom = Math.max(minzoom, Math.min(maxzoom, adjusted));
+        adjusted = getAdjusted(base, ratios, allowFloat);
 
-    return { center: center, zoom: zoom };
+    return {
+        center: merc.ll([(bl[0] + width / 2), (tr[1] + height / 2)], base),
```

Comment:
Would you be amenable to breaking this code up into a few more lines. While its nice to have it concise, in trying to grok this my eyes are struggling to parse the `(` and `[` to get a quick and correct sense of what object is getting passed into what function and what argument. So, while my suggestion is unrelated to the correctness of the code, I'm asking so I can review effectively without finding another cup of ☕

Reply:
You got it! Coming in a couple minutes :)

Reply:
Done! Hopefully it's a little easier to read now.

Reply:
Yes, perfect, thank you.